### PR TITLE
TAJO-1743: Improve calculation of intermediate table statistics

### PR DESCRIPTION
**Pull request by jinossy**

### CODE REVIEW

File: tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/RangeShuffleFileWriteExec.java [hunk 1]
```diff
@@ -56,14 +57,19 @@
   private KeyProjector keyProjector;
 
   public RangeShuffleFileWriteExec(final TaskAttemptContext context,
-                                   final PhysicalExec child, final Schema inSchema, final Schema outSchema,
-                                   final SortSpec[] sortSpecs) throws IOException {
-    super(context, inSchema, outSchema, child);
+                                   final ShuffleFileWriteNode plan,
```
Comment (blrunner) [inline tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/RangeShuffleFileWriteExec.java]: Could you explain why you remove output schema from parameters of the constructor?

**Comment by jinossy** [comment]
You can find it in PhysicalPlannerImpl. in-out schema is same in RangeShuffleWriter

**Comment by blrunner** [comment]
Thanks @jinossy 

I found following codes using your comments:
``` 
return new RangeShuffleFileWriteExec(ctx, subOp, plan.getInSchema(), plan.getInSchema(), sortSpecs); 
```

**Comment by blrunner** [comment]
+1

LGTM. :)

**Pull request closed without merging**